packages/web/integrations/axelar/types.ts

```diff
@@ -24,21 +24,23 @@ export type SourceChain =
   | "Fantom"
   | "Polygon"
   | "Moonbeam"
-  | "Moonbase"
+  | "Moonbase Alpha"
   | "Binance";
 
 /** Maps eth client chainIDs => axelar chain ids.
+ *
+ *  ethClientChainIDs must be specified in ../ethereuem/types.ts::ChainNames{} to map the name to a chainID, which is in turn used to add the network to EVM-compatible wallets, like Metamask.
+ *
+ * AxelarChainIds must be specified in SourceChain{} and are used in ./source-chain-configs.ts::SourceChainConfigs{} as <asset>::<network>::id values.
  *
  *  Values not included as keys are assumed to be the same across chainlist and Axelar.
  */
 export const EthClientChainIds_AxelarChainIdsMap: {
   [ethClientChainIds: string]: SourceChain;
 } = {
-  "Ropsten Test Network": "Ethereum",
```

Comment:
As mentioned in slack, we do need this to support MetaMask and Axelar on testnet

Reply:
Looks like Axelar accepts 'ethereum-2' for Goerli as that is now the network being used, not Ropsten.

Reply:
True. Ropsten is getting deprecated.